advance_ai_agents/finance_service_agent/routes/agentRoutes.py

```
@@ -0,0 +1,177 @@
+templates = Jinja2Templates(directory="templates")
+
+@router.get("/health", response_class=HTMLResponse)
+async def health_check(request: Request):
+    """Health check endpoint to verify the API server status and connections."""
+    try:
+        response_data = {
+            "status": "healthy",
+            "timestamp": datetime.datetime.now().isoformat(),
+            "uptime": "OK",
+            "api": {
+                "nebius_api": "connected" if NEBIUS_API_KEY else "not configured",
```

Ruff (0.11.9)

Undefined name `NEBIUS_API_KEY`

(F821)